dubbo-config/dubbo-config-api/src/test/java/org/apache/dubbo/config/integration/AbstractRegistryCenterExporterListener.java

```diff
@@ -56,7 +57,13 @@ public abstract class AbstractRegistryCenterExporterListener implements Exporter
     @Override
     public void exported(Exporter<?> exporter) throws RpcException {
         ListenerExporterWrapper listenerExporterWrapper = (ListenerExporterWrapper) exporter;
-        FilterChainBuilder.CallbackRegistrationInvoker callbackRegistrationInvoker = (FilterChainBuilder.CallbackRegistrationInvoker) listenerExporterWrapper.getInvoker();
+
+        Invoker invoker = listenerExporterWrapper.getInvoker();
+        if (!(invoker instanceof FilterChainBuilder.CallbackRegistrationInvoker)) {
+            exportedExporters.add(exporter);
+            return;
+        }
```

Comment:
What is this scene? Can you give me a note?

Reply:
This is for tesing.